examples/api/app/api/hello/route.ts

```diff
@@ -1,10 +1,18 @@
 import { headers } from "next/headers";
 
+import { getCloudflareContext } from "@opennextjs/cloudflare";
+
 export async function GET() {
-  // Note: we use headers just so that the route is not built as a static one
   const headersList = headers();
-  const sayHi = !!headersList.get("should-say-hi");
-  return new Response(sayHi ? "Hi World!" : "Hello World!");
+
+  const fromCloudflareContext = !!headersList.get("from-cloudflare-context");
+
+  if (!fromCloudflareContext) {
+    return new Response("Hello World!");
+  }
+
+  const { env } = await getCloudflareContext();
```

Comment:
nit: add a link to wrangler.toml?

Reply:
what do you mean a link? as an option to specify the toml file?

note that in production there is no toml file 😕

Reply:
Something like "// Retrieve the env defined in `wrangler.toml`"